lightning_hpo/controllers/notebook.py

```diff
@@ -66,7 +66,10 @@ def stop_notebook(self, config: StopNotebookConfig) -> str:
                 self.db.put(notebook.collect_model())
                 return f"The notebook `{config.notebook_name}` has been stopped."
             return f"The notebook `{config.notebook_name}` is already stopped."
-        return f"The notebook `{config.notebook_name}` doesn't exist. Found the following notebooks: {list(self.r)}."
+        names = list(self.r)
+        if not names:
+            return "You don't have any notebooks. Create a notebook with `lightning run notebook --name=my_name`."
```

Comment:
Should we have it at the start of the function to return the message?

Reply:
I preferred to have the case where it doesn't exist together. I think it is more readable.